dawn/test/unit-test/dawn/IIR/TestIIRSerializer.cpp

```diff
@@ -427,7 +429,9 @@ TEST_F(IIRSerializerTest, IIRTests) {
   stmt->setID(22);
   iir::Accesses stmtAccesses;
   iir::Extents extents(ast::Offsets{ast::cartesian});
+  iir::Extents undefinedExtents(iir::HorizontalExtent{ast::cartesian}, iir::UndefinedExtent{});
   stmtAccesses.addReadExtent(42, extents);
+  stmtAccesses.addReadExtent(43, undefinedExtents);
```

Comment:
Serialization/deserialization routines won't do checks on the validity of the IR, but this is invalid IIR (no field has been set 43 as access id). I suggest, instead, to add a separate example of `ExprStmt` which contains a vertical indirection access pattern, correctly crafted so that it doesn't invalidate the IIR.